src/content/blog/live-content-collections-deep-dive.mdx

```diff
@@ -0,0 +1,898 @@
+---
+title: 'Live Content Collections: A Deep Dive'
+description: "Explore Astro 5.10's experimental live content collections - bringing real-time data to your Astro sites with the same familiar API you know and love."
+publishDate: '2025-06-26'
+authors:
+  - matt
+lang: 'en'
+coverImage: '/src/content/blog/_images/solstice-festival/astro-solstice-festival-schedule-cover.webp'
+socialImage: '/src/content/blog/_images/solstice-festival/astro-solstice-festival-schedule-og.webp'
+---
+
+import BlogContentImage from '/src/components/BlogContentImage.astro';
+import Note from '/src/components/Note.astro';
+
+**Live content collections represent the next evolution of content in Astro, bringing real-time data capabilities to the familiar content collections API you already know and love.**
+
+Content collections in Astro have been through several stages of evolution. They were first launched as an incredibly easy and powerful way to manage structured content from files on disk. Initially supporting Markdown, MDX and JSOn files, they allow you to build blogs, documentation sites, and more with great developer experience and type-safe data. 
+
+With Astro 5.0, content collections were expanded into a full-fledged Content Layer that supports pluggable loaders for all kinds of data sources, including APIs, CMSs, and more.
```

Comment:
```suggestion
With Astro 5.0, content collections were expanded into a full-fledged Content Layer that supported pluggable loaders for all kinds of data sources, including APIs, CMSs, and more.
```
Kind of *feels* like this should also still be in the past tense in this sentence? (Even though it continues to today)